public/style.css

```diff
@@ -19,28 +19,35 @@ div {
 	padding: 35px;
 }
 
```

Comment:
I noticed in your stylesheet that you have set the property "display" of the div tags to "block", this is the default behavior of divs so there is no need for that style block!